tensorboard/webapp/plugins/BUILD

```diff
@@ -8,15 +8,12 @@ licenses(["notice"])  # Apache 2.0
 ng_module(
     name = "plugins",
     srcs = [
-        "plugins.component.ts",
+        "plugins.container.ts",
         "plugins.module.ts",
     ],
-    assets = [
-        "plugins.component.css",
-        "plugins.component.html",
-    ],
     deps = [
         "//tensorboard/webapp/core",
+        "//tensorboard/webapp/core/store",
```

Comment:
Many places are depending on both /core and /core/store.  Is the latter necessary here?  Or can we drop the former, if it's not using /core/actions, etc?

Reply:
The two deps provide different things: `core/store` provides the State, Reducer, and Selectors, while `core` provides only the NgModule (`CoreModule`).  It may be that some of these deps are unused, but I'd like to punt that cleanup to a later PR.

Reply:
Makes sense, thanks!